string set_name
---
bool success
